rustler_codegen/src/ex_struct.rs

```diff
@@ -82,8 +83,12 @@ pub fn gen_decoder(
             let ident = field.ident.as_ref().unwrap();
             let ident_str = ident.to_string();
             let atom_fun = Ident::new(&format!("atom_{}", ident_str), Span::call_site());
+            let error_message = format!("Could not decode field :{} on %{}{{}}", ident.to_string(), struct_name.to_string());
```

Comment:
This should probably be moved to inside the error case to avoid any penalty performance penalties

Reply:
Resolving, this happens at compile time, so we don't care! 😄